meetup/Cargo.toml

```diff
@@ -4,6 +4,14 @@ version = "0.1.0"
 publish = false
 edition = "2018"
 
+[lib]
```

Comment:
I didn't think this was needed if the source file was lib.rs

Reply:
So it looks like the lib definition is needed, but I can remove the [[bin]] section without any noticeable changes